.codecov.yml

```diff
@@ -8,5 +8,18 @@ coverage:
     patch:
       default:
         informational: true  # Treat coverage info as informational only
+flags:
+  test:  # Unit tests
+    paths:
+    - .*
+    carryforward: false
+  test_cygwin:  # Cygwin env
+    paths:
+    - .*
+    carryforward: false
+  integration-test:  # Integration tests
```

Comment:
@abravalheri hopefully, this will allow Codecov not to report coverage drop on PRs not running integration tests.

* https://docs.codecov.com/docs/github-5-carryforward-flags
* https://docs.codecov.com/docs/flags#carryforward-flags
* https://docs.codecov.com/docs/carryforward-flags